index.js

```diff
@@ -1,2 +1,3 @@
 export {version} from "./build/package";
 export {default as geoStream} from "./src/stream";
+export {default as geoLength} from "./src/length";
```

Comment:
Do you have a preferred order (like alphabetical or "dependency order") for exports?